src/pages/en/install/auto.md

```diff
@@ -33,18 +33,60 @@ yarn create astro
+- `Completely empty`: A template that just includes the bare minimium to get started.
+
+Use the arrow keys (up and down) to navigate to the template you want to install, then press return (enter) to submit.
+
+### Install Dependencies (optional)
+The wizard will offer to run an `install` command for you at this time, which is optional.
+
+> If you do not do so at this time, you will need to [install dependencies](/en/install/auto#2-install-dependencies) after the wizard has finished, before starting your project.
+
+### Install any Official Astro Integrations (optional)
+You will be given the option at this time to add any [additional UI frameworks](/en/core-concepts/framework-components) (React, Svelte, Vue, Solid, Preact, Lit) and then other Astro official integrations (Tailwind, Turbolinks, Partytown, Sitemap) by running `astro add --yes`
+
+To select which Astro integrations, if any, you would like to include in your project, use the arrow keys (up and down) to navigate and the space bar to toggle selected/unselected. You can select multiple items at once, or you can continue without selecting any items.
+
+
+When you are satisfied with your selection, press return (enter) to submit. 
+
+>🟢 Selected integrations will display a solid (green) circle. (All integrations are **unselected by default**.) 
+>
+>⚠️ If every item is unselected, then no integrations will be added at this time. 
+
+
+These integrations, and any [Astro community integrations](https://astro.build/integrations), can also be added later by following the instructions in our [integrations guide](/en/guides/integrations-guide).
 
-When the `create-astro` install wizard is complete, you should see some recommended instructions on your screen to follow that will help you complete setup and start your new project. 
+### See Your Astro Config File
 
-The only required step remaining is to install your project's dependencies using a package manager like npm:
+After selecting your integrations to add, you should see the following terminal message followed by **the entire contents of the `astro.config.mjs` file** that Astro will include with your project:
+
+```bash
+Astro will make the following changes to your config file:
+```
+
+At this time, you can verify that any integrations have been added to your project configuation, as well as any required `install` actions have run.
+
+
+### Initialize a .git Repository (optional)
+
+At the final step of the wizard, you will now be offered to initialize a git repository in your new directory, if you plan to use the tool [Git](https://git-scm.com/) in your project. This is optional.
+
+### Next Steps
+
+When the `create-astro` install wizard is complete, you should see some recommended instructions ("Next Steps") on your screen to follow that will help you complete setup and start your new project. (e.g. `cd my-astro-site && npm run dev`)
+
+## 2. Install Dependencies
```

Comment:
I’m not a huge fan of the fact that this appears as a super important **STEP 2**, but is likely to be unnecessary for (most?) people who installed dependencies while running the wizard. But I don’t really know what to do about it? Except maybe, remove this and don’t mention that the install step is optional in the wizard? What are the scenarios I’m missing where you’d really want to avoid having dependencies installed for you?

Reply:
I'm with you on that. I thought about (but didn't serious consider) MAKE SURE YOUR DEPS ARE INSTALLED as a Step 2. 😅 

It's obviously an artifact from when the wizard did NOT offer to install for you. And, given that it is an optional step in the wizard... it sort of does need to be mentioned, and mentioned in this order. (Maybe not as its own step, granted. But if it's part of step one, then step one is BIG and all the other steps are ... not!)

I could not mention that the install step is optional, but simply not mentioning it doesn't prevent someone from possibly ending up here without having any dependencies installed.  😄 

So... still thinking about this one! And, happy for more ideas!

Reply:
Absolutely — agree that the head-in-sand strategy may not always be effective 😆 

I think maybe you’re right that if this is the last sub-step of 1, it’s importance will be reduced.

Also maybe at that point it’s worth rethinking if we need the numbering at all?

Reply:
Ooh, I love out-of-the-box rethinks!

Sometimes I forget that I'm ALLOWED TO REWRITE THINGS... not just edit them up.  😅

Reply:
My brain can't handle un-numbering right now (and note to self, if we DO, then we have to go back to Manual install and NOT say "go to step 3"), so for this version anyway, I'm going to leave them numbered!

Prepared to revisit if people complain.